### PR TITLE
Feature: geolocation

### PR DESCRIPTION
**Pull request by rjmah1**
#### Summary of changes/features added:
- Owner and User geolocation based on if a given location is passed in.

#### Screenshots of UI changes (update with each new commit): 
- 
image

given location locked behind this being implemented

## Pull Request Checklist
- [ ] Unit test written?
- [ ] (**For later in project**) Integration tests written?
- [ X ] Java docs written?

### CODE REVIEW

File: client/gradle.properties [hunk 1]
```diff
@@ -16,4 +16,5 @@ org.gradle.jvmargs=-Xmx2048m
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
-android.enableJetifier=true
\ No newline at end of file
+android.enableJetifier=true
+MAPBOX_DOWNLOADS_TOKEN=sk.eyJ1IjoicmptYWgxIiwiYSI6ImNraHYwejh6dTA2bnAycnBnczVuYmN2eXIifQ.Qo37ZU-LD08wTnQJWYhRJg
```
Comment (AdityaHarvi) [inline client/gradle.properties:20]: Didn't even know this was a file

File: client/app/src/main/res/layout/activity_select_geopage.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    <Button
+        android:id="@+id/confirm_location_button"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="16dp"
+        android:background="@drawable/default_button"
+        android:text="@string/confirm_location"
```
Comment (AdityaHarvi) [inline client/app/src/main/res/layout/activity_select_geopage.xml:29]: Case sensitive plz

**Review by AdityaHarvi — CHANGES_REQUESTED** [review]
Literally dont even know whats going on with selectGeopage >: D

**Review by SamLojpur — APPROVED** [review]
Sorry, added some code to connect to db and remove a nonexistent use case. otherwise lgtm

**Review by AdityaHarvi — APPROVED** [review]

**Pull request merged**